pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "strawberry-graphql"
 packages = [ { include = "strawberry" } ]
-version = "0.44.3"
+version = "0.44.4"
```

Comment:
You don't need to bump the version manually here. The deployment pipeline will automatically increment the version number if you add a Release file.